config/config.toml

```diff
@@ -54,6 +54,7 @@ x_feature_route=false
 tenant_user=false
 dev_click_to_pay=false
 dev_recon_v2_product=false
+under_maintenance=""
```

Comment:
please rename the feature flag from under_maintainence to maintainence_alert keeping in mind the use case of this